R/deployApp.R

```diff
@@ -280,7 +280,35 @@ deployApp <- function(appDir = getwd(),
                      application$id), {
       bundlePath <- bundleApp(target$appName, appDir, appFiles,
                               appPrimaryDoc, assetTypeName, contentCategory)
-      bundle <- client$uploadApplication(application$id, bundlePath)
+
+      if(!isShinyapps(accountDetails)){
```

Comment:
Good point. I added spacing to this and the other if and else statements in the submission.